snap/snapcraft.yaml

```diff
@@ -5,8 +5,78 @@ summary: Snap for NVIDIA DCGM and DCGM exporter
 license: Apache-2.0
 contact: solutions-engineering@lists.canonical.com
 description: |
-  This snap includes NVIDIA DCGM and DCGM exporter to provide GPU monitoring via
-  Prometheus metrics. This snap does not include the dcgmproftester.
+  This snap includes NVIDIA DCGM and DCGM-exporter to monitor GPU via Prometheus metrics
+  A use can setup a Grafana dashboard to visualize the metrics.
+
+  The snap includes the following components:
+  - DCGM: Data Center GPU Manager
+  - DCGM-exporter: A Prometheus exporter for DCGM metrics
+  - NV-Hostengine: A service that communicates with the NVIDIA GPU driver to get the metrics
+
+  **Note**: The `DCGM-exporter` service is disabled by default because a user might not be interested
+  in collecting metrics as this is a DCGM snap, not the DCGM-exporter snap.
+
+  **Quick Start**
+  ---
+  Getting the NVIDIA GPU metrics would look like this:
+
+  ```
+  # Install the snap
+  sudo snap install dcgm
+
+  # Start the DCGM exporter service
+  sudo snap start dcgm.dcgm-exporter
+
+  # Get the metrics
+  curl localhost:9400/metrics
+  ```
+
+  **Config**
+  ---
+  The DCGM snap provides several configuration options through the `snap` command.
+
+  ```
+  # Get all the configuration options
+  sudo snap get dcgm
+
+  # Example of setting the NV-Hostengine port
+  sudo snap set dcgm nv-hostengine-port=5577
+
+  # Restart the NV-Hostengine service to apply the changes
+  sudo snap restart dcgm.nv-hostengine
+  ```
```

Comment:
this feels like another tutorial, "how to alter the nv-hostengine listening port", but we probably don't want to write mini-tutorials for every single option. How generalizing to "how to configure snap services"?